### ISSUE TITLE
Stacked column with shared tooltip doesn't always re-highlight series

### ISSUE THREAD

**Issue by jwickman** [highcharts/highcharts#13637]
#### Expected behaviour
In a stacked column chart with shared tooltip, when hovering over a column that has corresponding points for all series, all series should be fully opaque/highlighted.

#### Actual behaviour
When going from hovering over a column that does not have all of the data points to one that does, any series that were missing in the previous point get stuck as translucent, but only sometimes.

#### Live demo with steps to reproduce
https://jsfiddle.net/6aLgv0u1/1/

1. Start out hovering over the "Oranges" column at around y=11.5.
2. Move your mouse to the right such that it is in the space above the "Pears" column
3. Note that even though the tooltip is displaying all of the appropriate series data, those series are not all highlighted ("Joe" is dehighlighted/translucent).

![2020-06-04_07h27_21](https://user-images.githubusercontent.com/24733558/83752727-007fa980-a637-11ea-8597-da8f54b7415b.png)

#### Product version
Noticed in Highcharts 7.2.0, also tried in 8.1.0.

#### Affected browser(s)
Chrome 83 (haven't tested others)

**Comment by sebastianbochan** [comment]
Hi @jwickman,
Thank you for reporting. The experience is visible due to inactive state. When you disable it, all is highlighted proper.

Demo:
- https://jsfiddle.net/BlackLabel/z8b9xaqu/

Let me know if it worked properly.

**Comment by jwickman** [comment]
Hi @sebastianbochan,

Yes I am aware that that feature can be disabled as a workaround. However, the fact remains that it is a bug in the behavior of that feature when enabled, correct? It seems to work fine if hovering over the column, but when finding the nearest value, it doesn't reset the state.

**Comment by sebastianbochan** [comment]
Hi @jwickman,
Im close to your opinion.

@pawelfus what do you think?

**Comment by pawelfus** [comment]
I agree, it's a bug. When the previous stack contains points from all series it works fine: https://jsfiddle.net/BlackLabel/h563dsgc/
